### ISSUE TITLE
IFCObjectDefaults issue

### ISSUE THREAD

**Issue by barnabasmolnar** [xeokit/xeokit-sdk#276]
Hey there,

It seems like that the objectDefaults option isn't quite working as described here: https://github.com/xeokit/xeokit-sdk/wiki/Viewing-BIM-Models-Offline#setting-initial-states-for-ifc-types

![](https://cdn.discordapp.com/attachments/275761044167589889/686885335095181312/defaults.png)

```js
const model = {
  id: 'example-model',
  src: './xkt/duplex.xkt',
  metaModelSrc: './xkt/duplex.json',
  edges: true,
  objectDefaults: {
    DEFAULT: {
      colorize: [1, 1, 1],
    },
  },
};
```

This is with the duplex models from the xeokit examples section. The expectation here would be that all elements are white but as you can see on the screenshot above it has these weird pink and grayish colours.

I believe you may have already had an email exchange with @eriadam about this issue but I just wanted to log it here as well with an example.

**Comment by barnabasmolnar** [comment]
I had a look at https://github.com/xeokit/xeokit-sdk/issues/233 but the issue still seems open as far as the DEFAULT colorize option is concerned.

Tried it with the holter tower model like so:

```js
const holterTower = {
  id: 'holter-tower',
  src: './xkt/HolterTower.xkt',
  metaModelSrc: './xkt/HolterTower.json',
  edges: true,
  objectDefaults: {
    IfcSpace: {
      pickable: false,
      opacity: 0.2,
    },
    DEFAULT: {
      colorize: [1, 1, 1],
    },
  },
};
```

With the result being:
![](https://cdn.discordapp.com/attachments/275761044167589889/686893219979722791/holterTower.png)

**Comment by xeolabs** [comment]
Fixed in b3e6ce3
